ansible/examples/builder.yml

```diff
@@ -33,8 +33,10 @@
       tags:
         always
 
+    # get appropriate secret (clumsy variable names because you can't start
+    # a variable with a digit
     - set_fact:
-        token: "{{ jenkins_api_token }}"
+        token: "{{ ('2' in api_uri) | ternary(api_token_2_jenkins, api_token_jenkins)}}"
```

Comment:
Might it be cleaner to store these in individual host-specific `{{ secrets_path }}/{{ host_name }}/jenkins_api_token.yml` files?

Reply:
It's cleaner 😆.  IDK, I guess it doesn't matter if we never plan on having more than the 2 Jenkins instances.

Reply:
could also have hostname as a key in the dict I suppose.  Maybe having individual files would make it easier to edit in place etc. because more things handle the "straight encrypted file" than handle "encrypted strings".

Reply:
OK, I reworked it, and also removed slave.yml, a symlink I had from long ago that somehow sometime snuck back in

files are simple 'ansible-fault encrypt' now, named by hostname

Reply:
I like that better.  Thx!